lib/item.rb

```diff
@@ -1,10 +1,10 @@
 class Item
+  attr_accessor :name,
+                :description,
+                :unit_price,
+                :updated_at
```

Comment:
Great job using an accessor rather than having both a reader and writer for certain variables